config/initializers/ab_tests.rb

```diff
@@ -20,4 +20,15 @@ module AbTests
         0,
     },
   ).freeze
+
+  # This test is in place to allow a graceful transition from TrueID being the sole vendor to a
+  # multi-vendor configuration. This test will eventually replace the DOC_AUTH_VENDOR test.
+  SOCURE = AbTestBucket.new(
+    experiment_name: 'Socure',
+    buckets: {
+      socure: IdentityConfig.store.doc_auth_vendor_switching_enabled ?
+                          IdentityConfig.store.doc_auth_vendor_socure_percent :
+                          0,
+    }.compact,
+  ).freeze
```

Comment:
Heads up that I'm hoping to get #11026 landed soon--in that case, I'm guessing you'll want to add a block to this to determine the discriminator to use (I'm assuming the discriminator will be the document capture session uuid).

Reply:
Yup, I started to do this off your branch, but then decided to move on and update whenever that happens.